natural_language_processing/extractive_question_answering/bert_large/run_mlperf.py

```diff
@@ -43,6 +43,10 @@ def parse_args():
     parser.add_argument("--squad_path",
                         type=str,
                         help="path to directory with ImageNet validation images")
+    parser.add_argument("--fixed_input", action='store_true',
+                        help="truncate input to fixed shape")
+    parser.add_argument("--input_size", type=int, default=384,
```

Comment:
Input_size will only take effect when fixed_input is True. Therefore I suggest making a single arg out of this that both enabled fixed_input and specifies the size. Say --fixed_input_size.